product_variant_change_attribute_value/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{
+    "name": "Product Variant Change Attribute Value",
+    "version": "14.0.1.0.0",
+    "author": "Camptocamp SA, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/product-variant",
+    "license": "AGPL-3",
+    "category": "Product Variant",
+    "depends": ["sale", "account", "product"],
```

Comment:
As the models are not mandatory, module dependency is not needed. Tests are updated to avoid the dependency.
```suggestion
    "depends": ["product"],
```

Reply:
Thanks